static/docs/commands-reference/pipeline_show.md

```diff
@@ -116,7 +116,7 @@ instead of stages.
                 `--------------'
   ```
 
-  To navigate, use `W`, `A`, `S`, `D` keys.
+  To navigate, use arrows or `W`, `A`, `S`, `D` keys.
 
   To exit, press `Q`.
```

Comment:
Make it part of the previous paragraph. No need to split it. Use `>` to make it a note.